pyramid_mailer/mailer.py

```diff
@@ -16,6 +16,20 @@
 from pyramid_mailer._compat import SMTP_SSL
 
 
+def _check_bind_options(kw):
```

Comment:
I think I would argue that for consistency you should call `_check_bind_options` from `Mailer.bind` and separate the checking from the popping.